compiler/x/codegen/OMRCodeGenerator.hpp

```diff
@@ -223,6 +224,12 @@ struct TR_X86ProcessorInfo
    friend class OMR::X86::CodeGenerator;
 
    void initialize();
+   void assertUnused()
```

Comment:
Wouldn't a better name be something like `ensureRelocatableCompileSupport` or something like that? Could we just create functions to wrap the testAny for each of the feature flags and check the mask so we don't have to remove the assertUnused - we just have to add the flag to the mask?